Sources/PackageDescription4/SupportedPlatforms.swift

```diff
@@ -33,6 +33,10 @@ public struct Platform: Encodable {
 
     /// The Linux platform.
     public static let linux: Platform = Platform(name: "linux")
+
+    /// The Windows platform
+    @available(_PackageDescription, introduced: 5.2)
+    public static let windows: Platform = Platform(name: "windows")
```

Comment:
Can you add a trivial test for it? Here's an example: https://github.com/apple/swift-package-manager/blob/master/Tests/PackageLoadingTests/PD5LoadingTests.swift#L421

Reply:
Added!